guides/common/modules/proc_configuring-pull-based-transport-for-remote-execution.adoc

```diff
@@ -0,0 +1,44 @@
+[id="configuring-pull-based-transport-for-remote-execution_{context}"]
+= Configuring pull-based transport for remote execution
+
+By default, remote execution uses push-based SSH as the transport mechanism for the Script provider.
+If your infrastructure prohibits outgoing connections from {ProductName} to hosts, you can use remote execution with pull-based transport instead, because the host initiates the connection to {ProductName}.
+The use of pull-based transport is not limited to those infrastructures.
+
+The pull-based transport comprises `pull-mqtt` mode on {ProductName} in combination with a pull client running on hosts.
```

Comment:
```suggestion
The pull-based transport comprises `pull-mqtt` mode on {SmartProxies} in combination with a pull client running on hosts.
```
because you can configure it on a per-Smart Proxy basis.